### PR TITLE
feat: DAH-2649 add account already confirmed modal

### PR DESCRIPTION
**Pull request by tallulahkay**
## Description

Account already confirmed modal.

NOTE: Based off https://github.com/SFDigitalServices/sf-dahlia-web/pull/2425, merge after base pr is merged.

## Jira ticket

https://sfgovdt.jira.com/browse/DAH-2649

### Version Control

- [ ] branch name begins with `angular` if it contains updates to Angular code
- [x] branch name contains the Jira ticket number
- [x] PR name follows `type: TICKET-NUMBER Description` format, e.g. `feat: DAH-123 New Feature`

### Code quality

- [x] [the set of changes is small](https://google.github.io/eng-practices/review/developer/small-cls.html#what-is-small)
- [x] all automated code checks pass (linting, tests, coverage, etc.)
- [x] code irrelevant to the ticket is not modified e.g. changing indentation due to automated formatting
- [x] if the code changes the UI, it matches the UI design exactly
- [ ] if the changes include human translations, follow the [human translations process](https://sfgovdt.jira.com/l/cp/XS1KpvE4)

### Review instructions

- [x] instructions specify which environment(s) it applies to
- [x] instructions work for PA testers
- [x] instructions have already been performed at least once

### Request review

- [x] PR has `needs review` label
- [x] Use `Housing Eng` group to automatically assign reviewers, and/or assign specific engineers
- [ ] If time sensitive, notify engineers in Slack

### CODE REVIEW

File: app/javascript/authentication/SignInForm.tsx [hunk 1]
```diff
@@ -10,40 +10,106 @@ import {
...
-  // TODO(DAH-1575): Upgrade React-Hook-Form. Note: When you update to Version 7 of react-hook-form, "errors" becomes: "formState: { errors }""
-  // This is causing a linting issue with unbound-method, see open issue as of 10/21/2020:
-  // https://github.com/react-hook-form/react-hook-form/issues/2887
-  // eslint-disable-next-line @typescript-eslint/unbound-method
-  const { register, handleSubmit, errors, watch } = useForm()
-  const [requestError, setRequestError] = useState<string>()
-
-  const onSubmit = (data: { email: string; password: string }) => {
-    const { email, password } = data
+const AccountAlreadyConfirmedModal = ({
```
Comment (cade-exygy) [inline app/javascript/authentication/SignInForm.tsx:26]: **question (non-blocking):** Is it possible to pull these Modals into their own file?

And is there any opportunity for having all of these modals use a new "base" Modal component?

`SignInForm.tsx` is getting pretty long. I am okay with handling this cleanup next sprint, though.

Comment (tallulahkay) [reply app/javascript/authentication/SignInForm.tsx:26]: Yes!

Comment (tallulahkay) [reply app/javascript/authentication/SignInForm.tsx:26]: I'm going to wait until the base pr is done before making any more changes

Comment (tallulahkay) [reply app/javascript/authentication/SignInForm.tsx]: @cade-exygy I moved the modals to their own file!!

I tried to make a base modal component, but it actually made it way more complicated - I think the two modals are two dissimilar

Comment (cade-exygy) [reply app/javascript/authentication/SignInForm.tsx]: Awesome! and that makes sense, thanks for taking a look!

File: app/javascript/modules/listingDetails/ListingDetailsAdditionalInformation.tsx [hunk 2]
```diff
@@ -18,12 +18,14 @@ export const ListingDetailsAdditionalInformation = ({
 }: ListingDetailsAdditionalInformationProps) => {
   const { unleashFlag: isSalesFcfsEnabled } = useFeatureFlag("FCFS", false)
 
+  listing.Realtor_Commission_Unit = "test"
+  listing.Realtor_Commission_Amount = null
```
Comment (cade-exygy) [inline app/javascript/modules/listingDetails/ListingDetailsAdditionalInformation.tsx:22]: **nitpick (if-minor):** debugging clean up I think?

Comment (tallulahkay) [reply app/javascript/modules/listingDetails/ListingDetailsAdditionalInformation.tsx:22]: Thanks! missed that!

**Review by cade-exygy — APPROVED** [review]
Looks good!

**Pull request merged**